src/vmm/src/config.rs

```diff
@@ -26,6 +30,8 @@ impl fmt::Display for ConversionError {
             ParseKernel(ref s) => write!(f, "Invalid input for kernel: {}", s),
             ParseMemory(ref s) => write!(f, "Invalid input for memory: {}", s),
             ParseVcpus(ref s) => write!(f, "Invalid input for vCPUs: {}", s),
+            ParseNetwork(ref s) => write!(f, "Invalid input for network: {}", s),
+            ParseBlock(ref s) => write!(f, "Invalid input for network: {}", s),
```

Comment:
```suggestion
            ParseBlock(ref s) => write!(f, "Invalid input for block: {}", s),
```

Reply:
Thanks, will include this fix in the following PR.